### PR TITLE
Dynamic choice variable

### PR DESCRIPTION
**Pull request by liortct**
Addresses - #22 
This PR is forked from - https://github.com/cookiecutter/cookiecutter/pull/1768

### CODE REVIEW

File: tests/test_prompt.py [hunk 1]
```diff
@@ -115,6 +115,33 @@ def test_should_render_dict(self):
             'details': {'Slartibartfast': 'Slartibartfast'},
         }
 
+    def test_should_render_dynamic_choice_variable(self):
+        """
+        Verify dynamic choice variable, rendered correctly.
+
+        Added because issue #1774.
```
Comment (yuvaldolev) [inline tests/test_prompt.py]:
```suggestion
        Tests the fix for issue #1774.
```

File: cookieninja/prompt.py [hunk 2]
```diff
@@ -186,57 +198,53 @@ def prompt_for_config(context, no_input=False):
...
-                val = render_variable(env, raw, cookiecutter_dict)
-
-                if not no_input:
-                    val = read_user_variable(key, val)
-
-                cookiecutter_dict[key] = val
-        except UndefinedError as err:
-            msg = f"Unable to render variable '{key}'"
-            raise UndefinedVariableInTemplate(msg, err, context) from err
+        raw = rendered_raw
```
Comment (yuvaldolev) [inline cookieninja/prompt.py]: No need for this variable, as it's the same as `rendered_raw`, but used interchangeably in the if statements.
We can just use `rendered_raw` from here on.

**Review by yuvaldolev — CHANGES_REQUESTED** [review]

File: cookieninja/prompt.py (hunk 2 above)
Comment (liortct) [reply cookieninja/prompt.py]: Fixed

File: tests/test_prompt.py (hunk 1 above)
Comment (liortct) [reply tests/test_prompt.py]: Fixed

**Review by yuvaldolev — APPROVED** [review]
Awesome :)

**Pull request merged**